symplyphysics/definitions/linear_coefficient_of_thermal_expansion.py

```diff
@@ -35,27 +35,31 @@
 Linear :symbols:`thermal_expansion_coefficient` of the object.
 """
 
-length = clone_as_function(symbols.length, display_symbol="l(T, p)")
+temperature = symbols.temperature
 """
-:symbols:`length` of the object as a function of temperature
-and, indirectly, pressure :math:`p`.
+:symbols:`temperature` of the object.
 """
 
-temperature = symbols.temperature
+pressure = symbols.pressure
 """
-:symbols:`temperature` of the object.
+:symbols:`pressure` of the object, which is held constant.
+"""
+
+length = clone_as_function(symbols.length, [temperature, pressure])
+"""
+:symbols:`length` of the object as a function of temperature and, indirectly, pressure :math:`p`.
 """
 
 definition = Eq(
     linear_expansion_coefficient,
-    Derivative(length(temperature), temperature) / length(temperature),
+    Derivative(length(temperature, pressure), temperature) / length(temperature, pressure),
 )
 r"""
 :laws:symbol::
 
 .. only:: comment
 
-    Derivatives with one parameter held constant are not very pretty when autogenerated.
+    Derivatives with parameters held constant are not very pretty when autogenerated.
```

Comment:
Actually you can try to autogenerate it. It should look like a partial derivative.

Reply:
I want to try to make a prettier version of a derivative with fixed arguments, namely $`\left( \frac{\partial f}{\partial x} \right)_{y}`$, at least for now it doesn't look too difficult to me. If that turns out to be too daring a task, we can resort to the usual printer.

Reply:
Latex adds parameters to function so it should look like `df(x, y) / dx`. I think it's enough and we do not need to additionally specify constant parameter.